docs/astro/src/content/docs/guide/platforms/web.mdx

```diff
@@ -0,0 +1,101 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Web
+description: Running Slint on the Web with Wasm
+---
+
+While Slint is mainly designed for native applications (desktop, embedded, mobile),
+it can also run in a web browser using WebAssembly (Wasm).
+
+:::caution[Caution]
+Currently, only the Rust language is supported when targeting the web, and the only renderer available is FemtoVG.
+:::
+
+This renderer draws the UI into a `<canvas>` element on the page.
+The rendering happens entirely within the canvas, so Slint does not rely on the browser's DOM or CSS engine.
+This makes it possible to have a consistent look across platforms, but also introduces some limitations:
+
+* Slint handles its own font rendering, so it does not benefit from the browser's text rendering.
+* Accessibility features (such as screen readers) are not available.
+* The UI does not behave like a typical web application, since it doesn't use standard HTML elements.
+
+Because of these trade-offs, running Slint in the browser is currently not recommended for building general-purpose web applications.
+Instead, it is best suited for:
+
+* Demos and examples that can run directly in the browser without requiring installation.
+* Applications where the web is not the primary platform, but a consistent UI is still needed.
+* Tools or dashboards where native-style rendering is more important than web integration.
+
+In the future, we plan to explore a renderer that uses the DOM directly, but this is not currently the focus.
+
+## Building for WebAssembly
+
+To compile and run a Slint application in the browser, you will use
+[`wasm-bindgen`](https://rustwasm.github.io/docs/wasm-bindgen/) and [`wasm-pack`](https://rustwasm.github.io/wasm-pack/).
+
+The tutorial has a chapter for a step-by-step walkthrough.
```

Comment:
Since we're inside the same astro site, this really should be link to that chapter, IMO. Or at the very least to the tutorial.

(Also, unlike in German, for easier skimming it's great to mention the objective first ("For a walk-through"))

```suggestion
For a step-by-step walkthrough, check out the last chapter of the <Link type="quickstart" />.
```